changelog/front-commerce-3.2-2.28.mdx

```diff
@@ -0,0 +1,253 @@
+
+This release includes the following features:
+
+- all Magento1 modules are now available in the `@front-commerce/magento1`
+  package. Follow the
+  [Magento1 extension's documentation](/docs/3.x/extensions/magento1/) to learn
+  how to use it.
+- Adyen payment integration is also available in the `@front-commerce/adyen`
+  package
+- Akamai Image Manager integration is available in the
+  `@front-commerce/akamai-image-manager` package
+
+### Contribution mode and Magic Button
+
+[Six months ago, we introduced the Magic Button](/changelog/front-commerce-2.25#empowering-content-contribution-with-the-magic-button).
+A feature that improves the contribution experience and makes content editing
+more efficient.
+
+With Front-Commerce 3.x, we've revamped the technical foundations of the Magic
+Button to embrace Web standards and improve how it works. This new version is
+now available in Front-Commerce 3.2.
+
+It means that all supported extensions (Magento2, Contentful…) now have full
+feature-parity with Front-Commerce 2.x when it comes to identify storefront
+content, preview unpublished content, auto-refresh etc.
+
+### Server Services (unstable API)
+
+On a technical level, we've also introduced a new concept: **server Services**.
+
+**Why?** Until now, several Front-Commerce core features provided a way to
+register some custom logic using loaders. Examples:
+
+- `loaders.Sitemap.registerNodesFetcher`
+- `loaders.Page.registerUrlMatcher`
+- `loaders.ContentMetadataExtractorRegistry.register`
+
+It forced developers to manually create loaders when wanting to use these
+features outside of a GraphQL context, such as in an express middleware.
+
+**What?** Server Services is our proposal for relocating such responsibilities
+outside of "loaders", so that GraphQL loaders remains only responsible for
+fetching data from remote sources.
+
+**How?** We've introduced a lifecycle hook API that allows developers to
+register custom logic into core Services. Core Service are injected more
+globally, and can be used in GraphQL (context enhancers, loaders, resolvers) but
+also in application's routes as they're part of the `FrontCommerceApp` object.
+
+You can read
+[the current API design in our documentation](/docs/3.x/api-reference/front-commerce-core/defineExtension#unstable_lifecyclehooks).
+We'd love to hear your feedbacks on this, **please let us know how you feel
+about this!**
+
+### Other changes
+
+```mdx-code-block
+<details>
+  <summary><h3 className="mb-0">Performance improvements</h3></summary>
+```
+
+- Layout and root route never revalidate anymore by default, to prevent useless
+  extra HTTP requests
+
+```mdx-code-block
+</details>
+```
+
+```mdx-code-block
+<details>
+  <summary><h3 className="mb-0">Features</h3></summary>
+```
+
+- A feature flag has been added to check whether the `ordersFilters` feature is
+  supported by the backend in use.
+- We added a front-commerce script to check for unused SCSS variables and CSS
```

Comment:
```suggestion
- We added a front-commerce cli script to check for unused SCSS variables and CSS
```

Reply:
If https://github.com/front-commerce/developers.front-commerce.com/pull/826 were merged, we would be able to link to the cli documentation, or extract it to this MR? https://deploy-preview-826--heuristic-almeida-1a1f35.netlify.app/docs/3.x/api-reference/front-commerce-remix/#find-unused-style-variables

Reply:
I've added the link. Since we'll merge the other MRs soon after and can leave with a dead link for a few hours.